examples/web-sdk-sample/packages/vue2/src/components/UploadFile.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class='list row'>
+    <div class='col-md-12'>
+      <h4>Upload file to S3 using AWS SDK</h4>
+      <div class='flex-container'>
+        <input type='file' ref='fileInput' />
+        <button class='btn btn-sm btn-success' @click='uploadFile'>
+          Upload
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import { fromCognitoIdentityPool } from '@aws-sdk/credential-providers';
+
+const REGION = 'your bucket region';
```

Comment:
set this value to `.env` (env.example) file ?

Reply:
added to env.example